CRM/Contact/Import/Form/DataSource.php

```diff
@@ -168,63 +168,69 @@ public function setDefaultValues() {
    * Call the DataSource's postProcess method.
    *
    * @throws \CRM_Core_Exception
+   * @throws \API_Exception
    */
   public function postProcess() {
     $this->controller->resetPage('MapField');
+    if (!$this->getUserJobID()) {
+      $this->createUserJob();
+    }
+    else {
+      $this->updateUserJobMetadata('submitted_values', $this->getSubmittedValues());
+    }
+    // Setup the params array
+    $this->_params = $this->controller->exportValues($this->_name);
+
+    $storeParams = [
+      'onDuplicate' => $this->exportValue('onDuplicate'),
+      'dedupe' => $this->exportValue('dedupe'),
+      'contactType' => $this->exportValue('contactType'),
+      'contactSubType' => $this->exportValue('subType'),
+      'dateFormats' => $this->exportValue('dateFormats'),
+      'savedMapping' => $this->exportValue('savedMapping'),
+    ];
 
-    if (1) {
```

Comment:
This is noise - if I can rebase out https://github.com/civicrm/civicrm-core/pull/23253 will be an easier read